pkg/convert/xliff12.go

```diff
@@ -51,7 +51,7 @@ type context struct {
 
 // FromXliff12 converts serialized data from the XML data in the XLIFF 1.2 format into a model.Translation struct.
 // For now original param is ignored.
```

Comment:
The comment `// For now original param is ignored.` does not reflect implementation.